### PR TITLE
Webpackify lab

### PR DESCRIPTION
**Pull request by Lukeghenco**
@gj this lab is good to go now too. It has some really large changes with imports and exports as well as some big directory structure changes. The solution branch has been updated with all of these changes as well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,53 +1,38 @@
 {
-  "name": "react-props-and-state-lab",
+  "name": "react-initial-state-lab",
```
Comment (gj) [inline package.json]: `react-props-and-state-lab` or `react-initial-state-lab`?

Comment (gj) [inline package.json]: In `public/index.html`, `index.html`, and `README.md`, the former is used.

Comment (Lukeghenco) [inline package.json]: yea that should be react-props-and-state-lab

Comment (Lukeghenco) [inline package.json]: Yep all good to go now, thanks!!

**Comment by Lukeghenco** [comment]
Thanks for the review @gj

**Pull request merged**